themes/bootstrap3/templates/search/advanced/layout.phtml

```diff
@@ -166,7 +166,7 @@
       <?php endif; ?>
     </div>
 
-    <div class="<?=$this->layoutClass('sidebar')?>">
+    <div class="<?=$this->layoutClass('sidebar')?>" role="region" aria-label="<?=$this->transEsc("Search Tips")?>">
```

Comment:
Is "Search Tips" an appropriate label for this whole region? While there are search tips at the bottom, there's also potentially a set of filter-related controls which are functional. I'm concerned this label might cause people to totally skip over those controls.

Reply:
I am hesitant to introduce new language tokens as I only know 5 languages and barely speak 2 of them, so translating seems like a steep cost for me, this one fit the best but your concerns are more than valid. If no proper language token can be found the `role` attribute cannot be used

Reply:
Just a clarification that fortunately you don't have to manage all the languages yourself. :) It would be enough to introduce a new translation in English. You can of course include translations for other languages if you have them. I appreciate your thoughtfulness and agree that new translation strings should only be added when really needed, and I'm not implying that you can find a good label for this case.

Reply:
Just to second @EreMaijala's comments -- we'll accept language strings in any language(s) you're willing and able to provide, and we have volunteers who help translate into all the other languages. I definitely don't expect everyone to speak every language, and of course I'd prefer that people only submit translation strings they are very confident in. The experts will help with the rest! :-)